website/docs/language/globals.md

```diff
@@ -59,159 +59,151 @@ Because of the way time locks work, **a corresponding time lock needs to be adde
-bytes4 tx.hashtype
+bytes tx.outputs[i].lockingBytecode
 ```
 
-Represents the hashtype used for the generation of the sighash and transaction signature. Can be used to enforce that the spender uses a specific hashtype. See the [sighash docs][sighash-docs] for the implications of different hashtypes.
+Represents the locking bytecode (`scriptPubKey`) of a specific output.
 
-## Output instantiation
-One of the main use cases of covenants is enforcing transaction outputs (where money is sent and how much). To assist with enforcing these outputs, there is a number of `Output` objects that can be instantiated. These objects are then used in combination with `tx.hashOutputs` as shown in the [`tx.hashOutputs` section][tx.hashOutputs].
+## Constructing locking bytecode
+One of the main use cases of covenants is enforcing transaction outputs (where money is sent). To assist with enforcing these outputs, there is a number of `LockingBytecode` objects that can be instantiated. These locking bytecodes can then be compared to the locking bytecodes of transaction outputs.
 
 #### Example
 ```solidity
-bytes34 out1 = new OutputP2PKH(bytes8(10000), pkh);
-bytes32 out2 = new OutputP2SH(bytes8(10000), hash160(tx.bytecode));
-require(hash256(out1 + out2) == tx.hashOutputs);
+bytes25 lockingBytecode = new LockingBytecodeP2PKH(pkh);
+int value = 10000;
+require(tx.outputs[0].lockingBytecode == lockingBytecode);
+require(tx.outputs[0].value == value);
 ```
 
-### OutputP2PKH
+### LockingBytecodeP2PKH
 ```solidity
-new OutputP2PKH(bytes8 amount, bytes20 pkh): bytes34
+new LockingBytecodeP2PKH(bytes20 pkh): bytes25
 ```
 
-Creates new P2PKH output serialisation for an output sending `amount` to `pkh`.
+Creates new P2PKH locking bytecode for the public key hash `pkh`.
 
-### OutputP2SH
+### LockingBytecodeP2SH
 ```solidity
-new OutputP2SH(bytes8 amount, bytes20 scriptHash): bytes32
+new LockingBytecodeP2SH(bytes20 scriptHash): bytes23
 ```
 
-Creates new P2SH output serialisation for an output sending `amount` to `scriptHash`.
+Creates new P2SH locking bytecode for the script hash `scriptHash`.
 
-### OutputNullData
+### LockingBytecodeNullData
 ```solidity
-new OutputNullData(bytes[] chunks): bytes
+new LockingBytecodeNullData(bytes[] chunks): bytes
 ```
 
-Creates new OP_RETURN output serialisation for an output containing an OP_RETURN script with `chunks` as its data.
+Creates new OP_RETURN locking bytecode with `chunks` as its OP_RETURN data.
 
-[bip143]: https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki#specification
 [bip68]: https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki
-[sighash-docs]: https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/replay-protected-sighash.md#digest-algorithm
-
-[tx.time]: /docs/language/globals#txtime
-[tx.hashOutputs]: /docs/language/globals#txhashoutputs
-
 [withAge()]: /docs/sdk/transactions#withage
 [withTime()]: /docs/sdk/transactions#withtime
+[covenants-guid]: /docs/guides/covenants
```

Comment:
```suggestion
[covenants-guide]: /docs/guides/covenants
```
I assume this will fix the broken md rendering on [introspection guide](https://github.com/Bitcoin-com/cashscript/blob/3c7b51aac064ee326dba366ac71be98181c71759/website/docs/language/globals.md#introspection-variables)

Reply:
Thanks!